content/02.introduction.md

```diff
@@ -24,69 +24,59 @@ Further, we aim to amplify the true signal out of the vast noise produced by tho
+However, research in related viruses suggests that SARS-CoV-2 may possess some mechanisms that facilitate evasion of the antibody-mediated response.
+_In vitro_ analysis of nidoviruses including SARS-CoV suggests that these viruses can induce the development of double-membrane vesicles, preventing the dsRNA signature from being detected by the host immune system [@doi:10.1128/jvi.02501-05].
+This protective envelope can therefore insulate these coronaviruses from the innate immune response's detection mechanism [@doi:10.1016/j.jpha.2020.03.001].
+Presentation of the antigen is important because this step allows it to be detected by B-lymphocytes, also called B-cells, which are a type of white blood cell with antibodies bound to the membrane.
+Individual B-cells have unique membrane-bound antibodies.
+A B-cell is able to bind to an antigen only if its antibody is a match.
+For example, toll-like receptor 3 (TLR3) can detect double-stranded RNA (dsRNA) present during viral replication, allowing it to recognize a viral threat.
+The binding of a pattern recognition receptor (PRR), such as a TLR, to the antigen allows the B-cell to engulf it the virus via endocytocis.
+Once the antigen has been processed and presented on the cell surface, helper T-cells can bind to the MHC-II/antigen complex.
+This second binding step catalyzes the release of cytokines as part of the inflammatory response, as well as the proliferation of active B-cells (plasma cells) capable of producing the antibodies that respond to the antigen.
+Coronaviruses such as SARS-CoV are also able to evade the humoral immune response through other mechanisms, such as inhibiting certain cytokine pathways or down-regulating antigen presentation by the cells [@doi:10.1128/jvi.02501-05].
+Additionally, the cell will produce memory B-cells that can retain information about how to respond to future immune challenges by this pathogen.
+Understanding the humoral response to SARS-CoV-2 is important because this component of the immune response works to reduce the severity of infection in the short term and is also the mechanism through which the body can respond to reexposure.
+
+Research on other coronaviruses provides some evidence of how the innate immune response may proceed in the presence of SARS-CoV-2.
+Antibodies, also known as immunoglobulins, belong to five classes that describe which antigens they bind to.
+Two of the most common immunoglobulin classes are immunoglobulin M (IgM), which are the first antibodies produced in response to an infection, and immunoglobulin G (IgG), which are the most common antibodies found in the blood.
+When an antibody binds to an antigen, it forms a complex can then be cleared by the liver or spleen or through the complement cascade.
```

Comment:
Option B: You could skip the background lines of 69-71 & just go straight to the evidence related to SARS-CoV-2.
OR
Option A: a few errors to fix in lines 69-71. Reminder to fix these / for now temporary filler comment is that maybe something like this works:
Immunoglobulins belong to five classes which only differ in their constant regions (I need to look this up actually).
Two of the most common immunoglobulin classes are IgM, which are the first immunoglobulins produced in response to an infection, and IgG which are the most common antibodies found in the blood.
When an antibody binds to an antigen, it forms a complex that can then lead to several options (I need to look these up but I don't think liver is correct).